src/main/java/thePackmaster/cards/eurogamepack/Auctioneering.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.cards.eurogamepack;
+
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.powers.eurogamepack.VictoryPoints;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Auctioneering extends AbstractVPCard{
+    public static final String ID = makeID("Auctioneering");
+
+    public Auctioneering() {
+        super(ID, 1, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        this.baseMagicNumber = this.magicNumber = 4;
+        this.baseSecondMagic = this.secondMagic = 4;
+
+        this.baseBlock = block = Math.round(secondMagic/2);
+
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        if (!p.hasPower(VictoryPoints.POWER_ID)){this.applyToSelf(new VictoryPoints(AbstractDungeon.player, 0));}

```

Comment:
Instead of having this code in every card's use, you could apply it to the player at the start of combat instead.
For that you'd have to implement the OnBattleStartSubscriber in the main mod file and then apply this power in that method to the player